extensions/amp-a4a/0.1/amp-ad-type-defs.js

```diff
@@ -70,7 +83,7 @@ export class Renderer {
   /**
    * @param {!Object} unusedContext
    * @param {!Element} unusedContainerElement
-   * @param {!Object} unusedCreativeData
+   * @param {!CreativeData} unusedCreativeData
```

Comment:
Should we require`CreativeData` with `FriendlyFrameRenderer ` only?

Reply:
CreativeData will contain arbitrary data added by a validator, to be passed to the appropriate renderer. Once we add more renderers/validators, we'll adjust the type definition accordingly to include data that they might need to pass.

Reply:
I see. And we assume that there will only be three types of renderer in the near future? That individual ad network won't try to implement their own validator/renderer

Reply:
For the near future, I think we'll have just the 3 renderers, but I would not exclude the possibility of networks adding their own renderers and validators in the future should they have a good reason to do so.

Reply:
I am trying to avoid everyone adjust the type definition for `unusedCreativeData` here. Is it possible to override type definite with individual Renderer implementation or we have to declare all creativeData type with abstract function here?

Reply:
I think that might work. I've moved the definition over to friendly-frame-renderer.